src/main/java/com/amuzil/omegasource/magus/skill/forms/Forms.java

```diff
@@ -0,0 +1,33 @@
+package com.amuzil.omegasource.magus.skill.forms;
+
+import com.amuzil.omegasource.magus.skill.conditionals.InputData;
+import com.amuzil.omegasource.magus.skill.conditionals.key.KeyInput;
+import com.mojang.blaze3d.platform.InputConstants;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Forms {
+    // FORMS
+    public static final Form PUSH = new Form("push");
+    public static final Form PULL = new Form("pull");
+    public static final Form RAISE = new Form("raise");
+    public static final Form LOWER = new Form("lower");
+    public static final Form BURST = new Form("burst");
+    public static final Form ARC = new Form("arc");
+    public static final Form COMPRESS = new Form("compress");
+    public static final Form EXPAND = new Form("expand");
+    public static final Form TWIST = new Form("twist");
+    public static final Form SWIRL = new Form("swirl");
+    public static final Form STRIKE = new Form("strike");
+    public static final Form FORCE = new Form("force");
+    public static final Form BREATHE = new Form("breathe");
+    public static final Form STEP = new Form("step");
+
+
+    //These two may not be necessary.
```

Comment:
Don't add them for now then

Reply:
Kk, they're mainly for testing; I'm not sure we should even be registering input groups for forms right now, we may want to leave that up to each individual mod author.